### ISSUE TITLE
Use Title Case for friendly name attribute

### ISSUE THREAD

**Issue by ChrisRomp** [briis/weatherbit#22]
Example: San Francisco, CA

Result: the `friendly_name` attribute is "Weatherbit San francisco"
Expected: the `friendly_name` attribute should be "Weatherbit San Francisco"

I'm not a Python dev but I'll poke around and see if it's an obvious fix. Logging in any case.

**Comment by ChrisRomp** [comment]
Just to confirm it's not coming back incorrectly from Weatherbit, using this test call (GPS coords for SF city hall):

`https://api.weatherbit.io/v2.0/current?lat=37.7793&lon=-122.4192&key=[api key]`

Result, the `"city_name"` property looks correct:
```json
{
  "data": [
    {
      "rh": 87,
      "pod": "d",
      "lon": -122.42,
      "pres": 1012.7,
      "timezone": "America/Los_Angeles",
      "ob_time": "2020-07-30 15:20",
      "country_code": "US",
      "clouds": 100,
      "ts": 1596122400,
      "solar_rad": 51.3,
      "state_code": "CA",
      "city_name": "San Francisco",
      "wind_spd": 3.6,
      "wind_cdir_full": "west-northwest",
      "wind_cdir": "WNW",
      "slp": 1013.3,
      "vis": 5,
      "h_angle": -64.3,
      "sunset": "03:19",
      "dni": 649.39,
      "dewpt": 11.5,
      "snow": 0,
      "uv": 1.21286,
      "precip": 0,
      "wind_dir": 290,
      "sunrise": "13:12",
      "ghi": 284.92,
      "dhi": 73.89,
      "aqi": 27,
      "lat": 37.78,
      "weather": {
        "icon": "c02d",
        "code": "802",
        "description": "Scattered clouds"
      },
      "datetime": "2020-07-30:15",
      "temp": 13.7,
      "station": "1201P",
      "elev_angle": 19.58,
      "app_temp": 13.7
    }
  ],
  "count": 1
}
```

**Comment by briis** [comment]
I have a few other minor fixes, and I will fix this also.

**Comment by briis** [comment]
This is now fixed in release 0.31
